src/app/review-item/components/review-page/review-page.component.html

```diff
@@ -44,10 +44,59 @@ <h1>questionaire!</h1>
     </div>
     <div *ngIf="!showQuestionaire" class="case-details--accept-case">
         <h1>Die Fallbearbeitung</h1>
-        <div>Im Tatbestand siehst du alle für den Fall relevanten Informationen. Weiterer Hilfreicher Text...… Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore etLorem ipsum dolor sit amet, consetetur sadipscing elitr. Möchtest du diesen Fall annehmen?</div>
-        <div class="button-container">
-          <button class="accept-case-button case-button" (click)="accept()"><i class="fas fa-check"></i>&nbsp;Fall annehmen</button>
-          <a href='' class="reject-case-button case-button">Fall ablehnen</a>
+        <div *ngIf="!caseAccepted">
+          <div>Im Tatbestand siehst du alle für den Fall relevanten Informationen. Weiterer Hilfreicher Text...… Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore etLorem ipsum dolor sit amet, consetetur sadipscing elitr. Möchtest du diesen Fall annehmen?</div>
+          <div class="button-container">
+            <button class="accept-case-button case-button" (click)="accept()"><i class="fas fa-check"></i>&nbsp;Fall annehmen</button>
+            <a href='' class="reject-case-button case-button">Fall ablehnen</a>
+          </div>
+        </div>
+        <div *ngIf="caseAccepted">
+          <div *ngFor="let question of showQuestions; let index">
+            <app-question [question]="question" [index]="index" [questions]="questions"></app-question>
+          </div>
+          <div class="case-evaluation-comment-container case-process-container">
+            <p class="large-font">Fallbewertungskommentar</p>
+            <p class="explain">Möchtest du einen Kommentar zur Bewertung dieses Falls abgeben? Dieser wird nach Abschluss des Falls mitsamt deines Namens sichtbar sein.</p>
+            <div class="userinfo">
+              <div class="userprofile__bubble" style="background-color: #722ED1;">
+                J
```

Comment:
Why is there a "J" here?

I guess this should be the first letter of the users name?